app/task_routes.py

```diff
@@ -0,0 +1,113 @@
+from flask import Blueprint, request, jsonify, abort, make_response
+from app import db
+from app.models.task import Task
+from app.routes_helper import validate_item_by_id
+from datetime import datetime
+import os
+import requests
+
+# Define all routes with tasks_bp start with url_prefix (/tasks)
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+# Create a new task
+@tasks_bp.route("", methods=["POST"])
+def create_a_task():
+    
+    request_body = request.get_json()
+    
+    try:
+        new_task = Task(title=request_body["title"], description=request_body["description"])
+        
+    except:
+        return {
+            "details": "Invalid data"
+        }, 400
+    db.session.add(new_task)
+    db.session.commit()
```

Comment:
Great job using `db.session.<method-name>.` Session gives us access to the following:
* `db` is our app’s instance of SQLAlchemy.
* `session` represents our active database connection.
* By referencing db.session we can use SQLAlchemy’s methods to perform tasks like committing a change to a model, storing a new record of a model, and deleting a record.

By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the Goal model. ⭐